deploy/vagrant/provision/roles/ufs_gcs/files/config_alluxio.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+sed -i "s|^export ALLUXIO_UNDERFS_ADDRESS=.*|export ALLUXIO_UNDERFS_ADDRESS=gs://${GCS_BUCKET}|g" /alluxio/conf/alluxio-env.sh
```

Comment:
alluxio-env.sh contains the following line:

```export ALLUXIO_UNDERFS_ADDRESS=${ALLUXIO_UNDERFS_ADDRESS:-file://${ALLUXIO_HOME}/underFSStorage}```

this line will only set ALLUXIO_UNDERFS_ADDRESS if it is not already set ... that mean that the deploy script can simply run alluxio-env (or any other alluxio script) using an environment that has ALLUXIO_UNDERFS_ADDRESS set